src/Raven.Studio/typescript/commands/maintenance/saveAdminLogsSettingsCommand.ts

```diff
@@ -0,0 +1,25 @@
+import commandBase = require("commands/commandBase");
+import endpoints = require("endpoints");
+
+class saveAdminLogsSettingsCommand extends commandBase {
+    
+    constructor(private logMode: Sparrow.Logging.LogMode, private retentionTime: string, private retentionSize: Sparrow.Size, private compress: boolean) {
+        super();
+    }
+    
+    execute(): JQueryPromise<void> {
+        const url = endpoints.global.adminLogs.adminLogsConfiguration;
+        const payload = {
+            Mode: this.logMode,
+            RetentionTime: this.retentionTime,
+            RetentionSize: this.retentionSize,
+            Compress: this.compress
+        };
+
+        return this.post<void>(url, JSON.stringify(payload), null, { dataType: undefined })
+            .done(() => this.reportSuccess("Log Mode has been set to: " + this.logMode))
```

Comment:
success message is incorrect. This class allows us to configure not only log mode.

Reply:
=> Admin logs configuration was successfully set